### PR TITLE
287 display aggregate token balance

### LINKED ISSUE #287: Display aggregate token balance
## Overview
Display aggregate token balance

## Acceptance Criteria

- [ ]  Display sum of all tokens with fiat values as top balance
- [ ]  Display $0.00 if wallet is empty

### PR DESCRIPTION
**Pull request by donnyquixotic**
# Fixes #287

## Description
see issue

## Test scenarios

1. login with account with more than one fiat balance token
2. see sum of fiat values represented in header on wallet page
3. login with empty account (or account with no tokens having fiat values)
4. see $0.00

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have added appropriate test coverage

### CODE REVIEW

**Comment by Viterbo** [comment]
image

File: src/pages/evm/wallet/WalletPage.vue [hunk 1]
```diff
@@ -1,45 +1,34 @@
...
+
+const allTokens = computed(() => useBalancesStore().loggedBalances as EvmToken[]);
+const loadingStrings = computed(() => feedback.getLoadings);
+const loading = computed(() => feedback.isLoading('updateBalancesForAccount'));
+
+watch(allTokens, (newBalances: EvmToken[]) => {
+    let newFiatBalance = 0;
+    for (let balance of newBalances){
+        newFiatBalance += parseFloat(balance.fiatBalance);
+    }
```
Comment (Viterbo) [inline src/pages/evm/wallet/WalletPage.vue:22]: Most probably `balance.fiatBalance `is coming as `undefined` and that is causing the $NaN.
Ezra and I are working on the [new Token class PR](https://github.com/telosnetwork/telos-wallet/pull/286) to remove all those inconsistencies but for now you should fix this

```suggestion
    for (let balance of newBalances){
        newFiatBalance += parseFloat(balance.fiatBalance ?? 0);
    }
```

**Review by Viterbo — CHANGES_REQUESTED** [review]

File: src/pages/evm/wallet/WalletPage.vue (hunk 1 above)
Comment (donnyquixotic) [reply src/pages/evm/wallet/WalletPage.vue]: Pushed a fix checking that `fiatBalance` is defined before adding. This avoids unnecessarily parsing and adding 0 in all undefined cases.

**Review by Viterbo — APPROVED** [review]

**Review by EJSG — APPROVED** [review]
👌

**Pull request merged**